Benzy-Clubbers/benzy-clubbers-ember
# Added background and styles to reviews page

**What I did:**

- Added styling to each review
- Added a gradient background to reviews page

**How to Test**

1. Fetch and switch over to my branch. Pull from my branch to get the latest changes
2. Launch backend
3. `npm i or npm install`
4. ember serve
5. Under . /styles/_carddetailspage.scss
6. Feel free to mess around with styles
7. Make sure everything is work.
